core/src/main/java/com/linecorp/armeria/internal/client/ClientAttributeUtil.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.client;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.common.annotation.Nullable;
+
+import io.netty.util.AttributeKey;
+
+/**
+ * Contains helper methods used for armeria client internals.
+ */
+public final class ClientAttributeUtil {
+
+    private static final AttributeKey<PendingThrowableContainer> UNPROCESSED_PENDING_THROWABLE =
+            AttributeKey.valueOf(ClientAttributeUtil.class, "UNPROCESSED_PENDING_THROWABLE");
+
+    /**
+     * Sets a pending {@link Throwable} for the specified {@link ClientRequestContext}.
```

Comment:
Move this javadoc to the class description - add javadoc specific to this method